lib/src/theme/macos_theme.dart

```diff
@@ -249,19 +253,25 @@ class MacosThemeData with Diagnosticable {
     required this.dividerColor,
     required this.helpButtonTheme,
     required this.tooltipTheme,
+    required this.visualDensity,
   });
 
   /// A default light theme.
   factory MacosThemeData.light() =>
       MacosThemeData(brightness: Brightness.light);
 
   /// A default dark theme.
-  factory MacosThemeData.dark() => MacosThemeData(brightness: Brightness.dark);
+  factory MacosThemeData.dark() => MacosThemeData(
+        brightness: Brightness.dark,
+        visualDensity: VisualDensity.adaptivePlatformDensity,
```

Comment:
this line is useless because it's already being defined on line 228

Reply:
It's not, actually. This is a different scope.